client/src/components/Profile/Profile.js

```diff
@@ -0,0 +1,306 @@
+import React from "react";
+import "./Profile.css";
+import star from "./Images/star.png";
```

Comment:
Let's get the higher resolution star from Material Icons; there are resolution options where you click download